Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/CompositorWindowBase.cs

```diff
@@ -0,0 +1,275 @@
+
+        protected int renderFrameWidth;
+        protected int renderFrameHeight;
+        protected float aspect;
+        private float uiFrameWidth = 100;
+        private float uiFrameHeight = 100;
+
+        private const string trackingLostStatusMessage = "Tracking lost";
+        private const string trackingStalledStatusMessage = "No tracking update in over a second";
+        private const string locatingWorldAnchorStatusMessage = "Locating world anchor...";
+        private const string locatedWorldAnchorStatusMessage = "Located";
+        private const string calibrationLoadedMessage = "Loaded";
+        private const string calibrationNotLoadedMessage = "No camera calibration received";
+        private const int horizontalFrameRectangleMargin = 50;
+        protected const int textureRenderModeComposite = 0;
+        protected const int textureRenderModeSplit = 1;
+        private const float quadPadding = 4;
+
+        protected virtual void OnEnable()
+        {
+            renderFrameWidth = CompositionManager.GetVideoFrameWidth();
+            renderFrameHeight = CompositionManager.GetVideoFrameHeight();
+            aspect = ((float)renderFrameWidth) / renderFrameHeight;
+        }
+
+        protected void HolographicCameraNetworkConnectionGUI()
+        {
+            CompositionManager compositionManager = GetCompositionManager();
+            HolographicCameraNetworkManager cameraNetworkManager = GetHolographicCameraNetworkManager();
+
+            EditorGUILayout.BeginVertical("Box");
+            {
+                Color titleColor;
+                if (cameraNetworkManager != null &&
+                    cameraNetworkManager.IsConnected &&
+                    cameraNetworkManager.HasTracking &&
+                    cameraNetworkManager.IsAnchorLocated &&
+                    !cameraNetworkManager.IsTrackingStalled &&
+                    compositionManager != null &&
+                    compositionManager.IsCalibrationDataLoaded)
+                {
+                    titleColor = Color.green;
+                }
+                else
+                {
+                    titleColor = Color.yellow;
+                }
+                RenderTitle("Holographic Camera", titleColor);
+
+                if (cameraNetworkManager != null && cameraNetworkManager.IsConnected)
+                {
+                    if (cameraNetworkManager.ConnectedIPAddress == cameraNetworkManager.HoloLensIPAddress)
+                    {
+                        GUILayout.Label($"Connected to {cameraNetworkManager.HoloLensName} ({cameraNetworkManager.HoloLensIPAddress})");
+                    }
+                    else
+                    {
+                        GUILayout.Label($"Connected to {cameraNetworkManager.HoloLensName} ({cameraNetworkManager.ConnectedIPAddress} -> {cameraNetworkManager.HoloLensIPAddress})");
+                    }
+
+                    string anchorStatusMessage;
```

Comment:
>anchorStatusMessage [](start = 27, length = 19)

it may be worth renaming some of these "*Anchor*" items to be spatial coordinates.